apps/ledger-live-desktop/src/renderer/components/OnboardingAppInstall/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { Flex, Text, Button } from "@ledgerhq/react-ui";
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import { DeviceModelInfo } from "@ledgerhq/types-live";
+import { useTranslation } from "react-i18next";
+import { getDeviceModel } from "@ledgerhq/devices";
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+
+import DefaultAppsIllustration from "./DefaultAppsIllustration";
+import RestoreAppsIllustration from "./RestoreAppsIllustration";
+import CancelModal from "./CancelModal";
+import InstallSetOfApps from "./InstallSetOfApps";
+
+const fallbackDefaultAppsToInstall = ["Bitcoin", "Ethereum", "Polygon"];
+
+type Props = {
+  device?: Device | null | undefined;
+  restoreDevice?: DeviceModelInfo | null | undefined;
```

Comment:
don't use a verb for something that isn't a function (imo this is confusing)
`deviceToRestore` would be more suited.

Reply:
It would be nice to add a tsdoc comment also to explain what is this prop

Reply:
Done